ballerina-tests/tests/configuration_accept_encoding_test.bal

```diff
@@ -82,7 +78,7 @@ function testAcceptEncodingAuto() {
     http:Request req = new;
     req.setTextPayload("accept encoding test");
     http:Response|error response = acceptEncodingAutoEP->post("/", req);
```

Comment:
Shall we do as follows:
```ballerina
    http:Response response = check acceptEncodingAutoEP->post("/", req);
    // do the validations
```
Check for other places as well.

Reply:
> Shall we do as follows:
> 
> ```ballerina
>     http:Response response = check acceptEncodingAutoEP->post("/", req);
>     // do the validations
> ```
> 
> Check for other places as well.

Check my comment on your above suggestion. Since this is not an anti-pattern, kept the test cases as it is to handle errors